shardingsphere-infra/shardingsphere-infra-rewrite/src/main/java/org/apache/shardingsphere/infra/rewrite/sql/token/pojo/generic/SubstitutableColumnNameToken.java

```diff
@@ -76,11 +76,17 @@ public SubstitutableColumnNameToken(final int startIndex, final int stopIndex, f
     @Override
     public String toString(final RouteUnit routeUnit) {
         Map<String, String> logicAndActualTables = getLogicAndActualTables(routeUnit);
-        StringBuilder builder = lastColumn ? new StringBuilder(COLUMN_NAME_SPLITTER) : new StringBuilder();
+        StringBuilder result = new StringBuilder();
```

Comment:
How about the `java.util.StringJoiner`?

Reply:
I have tried StringJoiner， but the performance is not good enough.